azure/packages/azure-client/src/AzureClient.ts

```diff
@@ -210,11 +210,22 @@ export class AzureClient {
         };
 
         const codeLoader = { load };
+        const client: IClient = {
```

Comment:
For Tinylicious client, "write" was optional? Curious why difference between azure and tinyliciouv clients?

Reply:
I was under the impression that for Tinylicious, we wanted to allow a flag that attempts to start the connection in write mode, and leave the previous behavior otherwise, but for AzureClient, we decided to always try starting the connection in write mode

Reply:
We should look for consistency where possible. I think we have decided to adopt this approach for Tinylicious client  as well, so I will resolve this comment.